orchagent/vxlanorch.cpp

```diff
@@ -1405,6 +1406,21 @@ bool VxlanTunnelOrch::delOperation(const Request& request)
         return false;
     }
 
+    if (vtep_ptr == evpn_orch->getEVPNVtep())
+    {
+        for (auto it = vxlan_tunnel_table_.begin(); it != vxlan_tunnel_table_.end(); )
+	{
+            if ((it->second.get() != vtep_ptr) || (it->second->getDipTunnelCnt() != 0)  )
+	    {
+               SWSS_LOG_WARN("VTEP %s not deleted as there is user tuunel still in used", tunnel_name.c_str());
+	       return false;
+            }
+
+	    ++it;
```

Comment:
Please fix alignment in various places. Have this incr line in for-loop itself.

Reply:
Done